pg_migrations/0002_create_resource.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT  EXISTS public.resource
+(
+    id serial NOT NULL,
+    ori_id bigint NOT NULL,
+    iri character varying NOT NULL,
+    PRIMARY KEY (id),
+    UNIQUE (id),
+
+    CONSTRAINT had_primary_source FOREIGN KEY (iri)
+        REFERENCES public.source (iri) MATCH SIMPLE
```

Comment:
Shouldn't this reference the `id` of public.source instead of `iri`? Also, a better name for this foreign key could be something like `resource_id`.